docs-2.0/3.ngql-guide/7.general-query-statements/5.lookup.md

```diff
@@ -1,50 +1,57 @@
+
 - Count the number of vertices or edges with a tag or an edge type.
 
 ## OpenCypher compatibility
 
-This page applies to native nGQL only.
+This topic applies to native nGQL only.
 
 ## Prerequisites
 
-Before using the `LOOKUP` statement, make sure that relative indexes are created. For how to create indexes, see [CREATE INDEX](../14.native-index-statements/1.create-native-index.md).
+Before using the `LOOKUP` statement, make sure that at least one index is created. If there are already related vertices, edges, or properties before an index is created, the user must [rebuild native index](../14.native-index-statements/4.rebuild-native-index.md) after creating the index to make it valid.
+
+!!! caution
+
+    Indexes will weaken write performance for (90% or even more). Do not use indexes in a production environment. Please do not use indexes in a production environment at will, unless you know the impact of using indexes on the business.
 
 ## Syntax
 
 ```ngql
-LOOKUP ON {<vertex_tag> | <edge_type>} [WHERE <expression> [AND <expression> ...]] [YIELD <return_list>]
+LOOKUP ON {<vertex_tag> | <edge_type>} [WHERE <expression> [AND <expression> ...]] [YIELD <return_list>];
 
 <return_list>
-    <prop_name> [AS <col_alias>] [, <prop_name> [AS <prop_alias>] ...]
+    <prop_name> [AS <col_alias>] [, <prop_name> [AS <prop_alias>] ...];
 ```
 
-- The `WHERE` clause filters data with the specified conditions. Both `AND` and `OR` are supported between different expressions. For more information, see [WHERE](../8.clauses-and-options/where.md).
-- The `YIELD` clause specifies the results to be returned and the format of the results.
+- `WHERE <expression>`: filters data with specified conditions. Both `AND` and `OR` are supported between different expressions. For more information, see [WHERE](../8.clauses-and-options/where.md).
+
+- `YIELD <return_list>`: specifies the results to be returned and the format of the results.
+
 - If there is a `WHERE` clause but no `YIELD` clause:
-  - The Vertex ID is returned when `LOOKUP` a tag.
-  - The source vertex ID, destination vertex ID, and rank of the edge is returned when `LOOKUP` an edge type.
+  - The Vertex ID is returned when you `LOOKUP` a tag.
+  - The source vertex ID, destination vertex ID, and rank of the edge are returned when `LOOKUP` an edge type.
 
 ## Limitations of using `WHERE` in `LOOKUP`
 
 The `WHERE` clause in a `LOOKUP` statement does not support the following operations:
 
 - `$-` and `$^`.
-- In relational expressions, expressions with field names on both sides of the operator are not supported, such as `tagName.prop1> tagName.prop2`.
+- In relational expressions, expressions with field names on both sides of the operator are not supported, such as `tagName.prop1 > tagName.prop2`.
 - Nested AliasProp expressions in operation expressions and function expressions are not supported.
 - Range scan is not supported in the string-type index.
```

Comment:
This is supported. To verify, execute `PROFILE lookup on player where player.name > "A"`